### ISSUE TITLE
frontend is broken

### ISSUE THREAD

**Issue by celesteking** [Fraudmarc/fraudmarc-ce#22]
I've got a couple of reports processed into the DB, but frontend is basically useless.

I'm getting a blank screen with errors thrown in console:

```
vendor.0360ecc….js:1 ERROR TypeError: Cannot read property 'length' of null
    at Object.updateDirectives (main.f1a4648….js:1)
    at Object.updateDirectives (vendor.0360ecc….js:1)
    at $a (vendor.0360ecc….js:1)
    at rs (vendor.0360ecc….js:1)
    at ns (vendor.0360ecc….js:1)
    at $a (vendor.0360ecc….js:1)
    at rs (vendor.0360ecc….js:1)
    at is (vendor.0360ecc….js:1)
    at $a (vendor.0360ecc….js:1)
    at rs (vendor.0360ecc….js:1)
vendor.0360ecc….js:1 ERROR TypeError: Cannot read property 'length' of null
    at Object.updateDirectives (main.f1a4648….js:1)
    at Object.updateDirectives (vendor.0360ecc….js:1)
    at $a (vendor.0360ecc….js:1)
    at rs (vendor.0360ecc….js:1)
    at ns (vendor.0360ecc….js:1)
    at $a (vendor.0360ecc….js:1)
    at rs (vendor.0360ecc….js:1)
    at is (vendor.0360ecc….js:1)
    at $a (vendor.0360ecc….js:1)
    at rs (vendor.0360ecc….js:1)
vendor.0360ecc….js:1 ERROR TypeError: Cannot read property 'length' of null
    at e._next (main.f1a4648….js:1)
    at e.__tryOrUnsub (vendor.0360ecc….js:1)
    at e.next (vendor.0360ecc….js:1)
    at e._next (vendor.0360ecc….js:1)
    at e.next (vendor.0360ecc….js:1)
    at e._next (vendor.0360ecc….js:1)
    at e.next (vendor.0360ecc….js:1)
    at e._next (vendor.0360ecc….js:1)
    at e.next (vendor.0360ecc….js:1)
    at e.notifyNext (vendor.0360ecc….js:1)
```

​When trying to search for domain, I get: 
`No Domains. Get started by adding a domain.`
Along with following new error in console:

```
ERROR TypeError: Cannot read property 'forEach' of null
    at l.doFilter (main.f1a4648….js:1)
    at l.onFilter (main.f1a4648….js:1)
    at Object.handleEvent (main.f1a4648….js:1)
    at Object.handleEvent (vendor.0360ecc….js:1)
    at Object.handleEvent (vendor.0360ecc….js:1)
    at Wr (vendor.0360ecc….js:1)
    at vendor.0360ecc….js:1
    at e.o [as _next] (vendor.0360ecc….js:1)
    at e.__tryOrUnsub (vendor.0360ecc….js:1)
    at e.next (vendor.0360ecc….js:1)
```

There's nowhere to add the domain, no buttons to press and no mention of that in the docs.

You should've really added proper error handling out there. This software is unripe to use.

**Comment by celesteking** [comment]
This is the DB state currently, listing # of records:
```
 schemaname |         relname         | n_live_tup 
------------+-------------------------+------------
 public     | AggregateReportRecord   |          9
 public     | SpfAuthResult           |          9
 public     | DkimAuthResult          |          9
 public     | AggregateReport         |          3
 public     | dmarc_reporting_entries |          0
 public     | PoReason                |          0
```

**Comment by kcolemangt** [comment]
This issue has been resolved by the changes in PR #71, which significantly streamlines the build and deployment process, reducing setup time to just a few minutes.
